pkg/codegen/codegen.go

```diff
@@ -516,7 +554,7 @@ func GenerateEnums(t *template.Template, types []TypeDefinition) (string, error)
 		}
 	}
 
-	return GenerateTemplates([]string{"constants.tmpl"}, t, c)
+	return GenerateTemplates([]string{"enum-values.tmpl"}, t, c)
```

Comment:
Similar here, if the enum-values template wont need any other values, might be worth calling directly with EnumDefinitions, unless it may potentially have other values down the line